### ISSUE TITLE
Release 2.1.0 has invalid main reference in package.json

### ISSUE THREAD

**Issue by sudipto-ukg** [flexdinesh/browser-or-node#22]
The latest release's package.json `main` refers to `./lib/index.js`, but `lib` doesn't exist:

```
(node:82252) UnhandledPromiseRejectionWarning: Error: Cannot find module '/Users/sudipto/Source/abcd/node_modules/browser-or-node/lib/index.js'. Please verify that the package.json has a valid "main" entry
```

**Comment by FSteinhansesLT** [comment]
Yes, I have the same issue. I also needed to downgrade to 2.0.0 for now.

**Comment by eltharynd** [comment]
Same. broke my mongoose app :)

**Comment by Fgerthoffert** [comment]
Same over here, `Cannot find module '/Users/someuser/.nvm/versions/node/v18.12.1/lib/node_modules/@jahia/jahia-reporter/node_modules/browser-or-node/lib/index.js'. Please verify that the package.json has a valid "main" entry`.

```
➜  browser-or-node git:(v0.39.1) pwd
/Users/someuser/.nvm/versions/node/v18.12.1/lib/node_modules/@jahia/jahia-reporter/node_modules/browser-or-node
➜  browser-or-node git:(v0.39.1) ll
total 24
-rw-r--r--  1 someuser  staff   1.1K Dec  1 17:58 LICENSE
-rw-r--r--  1 someuser  staff   1.2K Dec  1 17:58 README.md
-rw-r--r--  1 someuser  staff   1.9K Dec  1 17:58 package.json
drwxr-xr-x  4 someuser  staff   128B Dec  1 17:58 src
```

The built version of the module is missing from the package

**Comment by flexdinesh** [comment]
Thanks for report everyone. I gotta stop publishing from my machine and setup GH actions to automate publishing. Long story short I merged a PR #21 yesterday and published a minor. But the `prepublish` script somehow did not run probably because I did not install the deps using `yarn`. 🤦 

Anyhoo I've published a patch now `v.2.1.1` and it should fix the missing `lib` issue. I double checked it after publish and made sure `lib` is in the published package - [Docsmill browser-or-node@2.1.1](https://docsmill.dev/src/browser-or-node@2.1.1). Sorry and thanks!
